conda_smithy/templates/run_docker_build.sh.tmpl

```diff
@@ -29,6 +29,9 @@ if [ -z "$CONFIG" ]; then
     exit 1
 fi
 
+pip install shyaml
```

Comment:
Would it make sense to add this to `conda-forge`?

xref: https://github.com/conda-forge/staged-recipes/issues/6557